tests/test_layers.py

```diff
@@ -454,6 +454,13 @@ def test_upsample(self):
         layer = keras.layers.UpSampling2D(size=(2, 3), data_format='channels_last')
         self._misc_conv_helper(layer, ishape)
 
+    @unittest.skipIf(StrictVersion(keras.__version__) < StrictVersion("2.2.3"),
+                     "The interpolation keyword not supported in Keras prior to 2.2.3.")
+    def test_upsample_bilinear(self):
```

Comment:
Let's still put this in test_upsample, just use 
if StrictVersion(keras.__version__) < StrictVersion("2.2.3"):
       layer = keras.layers.UpSampling2D(size=(2, 3), data_format='channels_last', interpolation='bilinear')
       self._misc_conv_helper(layer, ishape)